server/src/main/scala/de/frosner/broccoli/nomad/NomadModule.scala

```diff
@@ -37,5 +38,5 @@ class NomadModule extends AbstractModule with ScalaModule {
   @Provides
   @Singleton
   def provideNomadClient(config: NomadConfiguration, wsClient: WSClient): NomadClient =
-    new NomadHttpClient(config.url, wsClient)(play.api.libs.concurrent.Execution.defaultContext)
+    new NomadHttpClient(Uri.parse(config.url), wsClient)(play.api.libs.concurrent.Execution.defaultContext)
```

Comment:
Does this change resolve the problem reported in https://github.com/FRosner/cluster-broccoli/issues/81?